roles/aci/tasks/main.yml

```diff
@@ -19,10 +19,12 @@
   with_items: "{{ annotate_namespace_list }}"
   delegate_to: "{{ groups.oo_first_master.0 }}"
 - name: Get aci policy tenant name
-  shell: "grep -m1 'aci-policy-tenant' {{ aci_deployment_yaml_file }} | awk -F '\"' '{print $4}'"
+  shell: grep -m1 'aci-policy-tenant' {{ aci_deployment_yaml_file }} | awk -F '\"' '{print $4}'
```

Comment:
Can this `shell` be switched to `command`?

Reply:
Done.

Reply:
Same command using `command` module did not work hence I am using `shell` module.